homeassistant/components/config/auth_provider_homeassistant.py

```diff
@@ -23,7 +23,6 @@
 WS_TYPE_CHANGE_PASSWORD = 'config/auth_provider/homeassistant/change_password'
 SCHEMA_WS_CHANGE_PASSWORD = websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend({
     vol.Required('type'): WS_TYPE_CHANGE_PASSWORD,
-    vol.Required('current_password'): str,
```

Comment:
You misunderstood. I think that this is great 👍

Reply:
Oh oops 🤦‍♂️ I'll add it back